config/service_test_utils.py

```
@@ -0,0 +1,383 @@
+        try:
+            for service_name, service_utils in self.services.items():
+                session_manager = service_utils.db_manager.isolated_session(
+                    f"{ecosystem_test_id}_{service_name}"
+                )
+                sessions[service_name] = session_manager.__enter__()
+
+            yield sessions
+
+        finally:
+            # Cleanup sessions
+            for service_name, session in sessions.items():
```

Ruff (0.13.3)

Loop control variable `session` not used within loop body

Rename unused `session` to `_session`

(B007)